dags/dag_maintenance.py

```diff
@@ -121,18 +121,18 @@ def get_max_data_cleanup_configuration_function(**context):
     context["ti"].xcom_push(key="max_date", value=max_date.isoformat())
 
 
-get_configuration = PythonOperator(
+GET_CONFIGURATION = PythonOperator(
```

Comment:
suggestion: what I ended up doing was to move the code to a `create_dag` function so that these variables become local variables. Because it doesn't seem that these are intended to be "constants".